src/components/sucessfullUserRegisterModal.vue

```diff
@@ -0,0 +1,26 @@
+<template>
```

Comment:
can you create this component dynamic, so it can handle all type of successful response @ShivanshSinghnnovative